mypy/types.py

```diff
@@ -384,6 +384,7 @@ class Instance(Type):
 
     def __init__(self, typ: Optional[mypy.nodes.TypeInfo], args: List[Type],
                  line: int = -1, column: int = -1, erased: bool = False) -> None:
+        assert(typ is None or typ.fullname() not in ["builtins.Any", "typing.Any"])
```

Comment:
Yes we should, but these somehow got created in the tests. I have avoided this (` or fullname() == 'builtins'` above), and added the assertions here to verify this is indeed the case. Perhaps we can remove the tests now.

As Guido said above - or at least as I understand hes comment - it will be better not to define Any in builtins fixture at all.